### PR TITLE
Issue151

### PR DESCRIPTION
**Pull request by Bryar-Frank**
fixes issue #151

### CODE REVIEW

**Comment by bnmnetp** [comment]
this seems to duplicate much of #145 ??

**Comment by bnmnetp** [comment]
Now it is clear that this is a duplicate of #145

File: _sources/Selection/BooleanValuesandBooleanExpressions.rst [hunk 1]
```diff
@@ -63,7 +63,13 @@ two values are equal to one another.
...
     print(5 == 5)
+
     print(5 == 6)
+    
+    j = "hel"
+    print(j + "lo" == "hello")
+
+    
 
 In the first statement, the two operands are equal, so the expression evaluates
```
Comment (bnmnetp) [inline _sources/Selection/BooleanValuesandBooleanExpressions.rst:74]: I think a sentence or two explaining the third example and emphasizing the point of the third example is that python evaluates each side of the `==` statement before it tests for equality is super important. @pearcej do you agree?

Comment (Bryar-Frank) [reply _sources/Selection/BooleanValuesandBooleanExpressions.rst:74]: I have reworded the active code to remove order of operation confusion and added comments about the third question below

Comment (pearcej) [reply _sources/Selection/BooleanValuesandBooleanExpressions.rst:74]: Yes, and I think this rewording addresses the issue.

File: _sources/Selection/BooleanValuesandBooleanExpressions.rst [hunk 2]
```diff
@@ -63,10 +63,18 @@ two values are equal to one another.
...
     print(5 == 6)
+    
+    j = "hel" + "lo"
+    k = "hello"
+    print(j == k)
+    
 
 In the first statement, the two operands are equal, so the expression evaluates
 to ``True``.  In the second statement, 5 is not equal to 6, so we get ``False``.
+In the third statement, we see that concatenation in j wil result in the same value
```
Comment (bnmnetp) [inline _sources/Selection/BooleanValuesandBooleanExpressions.rst:76]: typo: missing an l in will

**Review by bnmnetp — CHANGES_REQUESTED** [review]

File: _sources/Selection/BooleanValuesandBooleanExpressions.rst (hunk 2 above)
Comment (Bryar-Frank) [reply _sources/Selection/BooleanValuesandBooleanExpressions.rst]: I apologize, I should have had someone review it first. I have fixed the misspelling.

**Comment by Bryar-Frank** [comment]
The misspelling has been fixed on this pull request.
@bnmnetp

**Pull request closed without merging**